### ISSUE TITLE
Experiment results: Relative change CI for negative revenue metrics

### ISSUE THREAD

**Issue by razvanpapadopol** [Automattic/abacus#793]
When testing out some examples for the exploration to use relative change to calculate impact, I've noticed something that may seem odd when looking at the absolute & relative results for a revenue metric with negative value such as "refunds".

<img width="495" alt="Screen Shot 2022-10-27 at 23 55 10" src="https://user-images.githubusercontent.com/14192054/198396628-77f3775b-d4ba-42e7-8682-d2c7b79befa3.png">

In this case, the "lift" on the right should actually be "loss reduction".

### To do

- [ ] 
- [ ]

**Comment by aaronyan** [comment]
Good catch on making the three values consistent for metrics where `higher is better: no`. I'm not sure what we should do, but I think making the experience more consistent and easier to interpret is a good direction!

What do you think about using terms like "earned" and "saved" so experimenters can expect to only see positive values as a good thing? For a `higher is better: no` metric, we would then see "saved":

- ACPU saved: -0.04 to +0.08 USD 
- Impact saved: -85k to +195k USD per month
- Lift: -26% to +75%

We would then have left-to-right be consistent between metrics to mean bad-to-good in the diagram and all three credible intervals.

**Comment by razvanpapadopol** [comment]
Using specific impact terms like "earned" and "saved" seems like a good idea for some kind of summary or conclusion (maybe related to https://github.com/Automattic/abacus/issues/794). And they would also highlight the value of `higher is better` property of a metric which it's not displayed in experiment results at the moment (looks like a UI bug!).

But this current issue is referring only to the "Lift" of a "Cash Sales" metric which is defined as `higher is better: yes` but its values are all negative. So the term would be 'earned', the absolute CIs are already showing "bad-to-good" as seen in my screenshot but the relative CI is showing "good-to-bad". 

I'm starting to believe this is not something we can fix on the client-side because of how the components are set to always display `"...{*.bottom_95} to {*.top_95}..."`. So in order to display the relative change of a negative earn "+74.39% to -26.2%", we should maybe update the `ratios` object in the API response to be `bottom_95: +74.39%` and `top_95: -26.2%`. WDYT, @jessie-ross ?

**Comment by jessie-ross** [comment]
> I'm starting to believe this is not something we can fix on the client-side because of how the components are set to always display "...{*.bottom_95} to {*.top_95}...". So in order to display the relative change of a negative earn "+74.39% to -26.2%", we should maybe update the ratios object in the API response to be bottom_95: +74.39% and top_95: -26.2%. WDYT, @jessie-ross ?

I can see that it would be easier to fix on the server-side, but I think it is entirely a client-side concern - the display of data. We have the most flexibility if we keep everything client-side, and we also keep the back-end away from a lot of complexity.

If we do it in the front-end the other option is switching around the control/treatment variation ids in the id string - both orders will exist on the ratios object :-)

I'm not too fussed about whether we switch the order around or not, I think it might be confusing either way. The most important part is highlighting which direction is better in some way. "earned" and "saved" is one such way, I'm not 100% on that terminology.

**Comment by razvanpapadopol** [comment]
Thanks for your input, Jessie!

> If we do it in the front-end the other option is switching around the control/treatment variation ids in the id string - both orders will exist on the ratios object

Switching the base / change ids for this specific case is definitely worth trying out since it means leaving the pure UI components untouched 😌 

> I think it might be confusing either way. 

Yeah, the confusion will still be there but at least we achieve some level of consistency by displaying the same direction (like "bad-to-good") on all columns of the metric assignment results.
